tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1061,80 +1061,50 @@ const docs = [
         data: new Buffer(xml).toString('base64')
       }
     }
-  },
-  {
-    _id: 'c49385b3594af7025ef097114104ef48',
-    reported_date: 1469578114543,
-    notes: '',
-    contact: {
-      _id: contactId,
-      name: 'Jack',
-      date_of_birth: '',
-      phone: '+64274444444',
-      alternate_phone: '',
-      notes: '',
-      type: 'person',
-      reported_date: 1478469976421
-    },
-    name: 'Number three district',
-    external_id: '',
-    type: 'district_hospital'
-  },
-  {
-    _id: contactId,
-    name: 'Jack',
-    date_of_birth: '',
-    phone: '+64274444444',
-    alternate_phone: '',
-    notes: '',
-    type: 'person',
-    reported_date: 1478469976421,
-    parent: {
-      _id: 'c49385b3594af7025ef097114104ef48',
-      reported_date: 1469578114543,
-      notes: '',
-      contact: {
-        _id: contactId,
-        name: 'Jack',
-        date_of_birth: '',
-        phone: '+64274444444',
-        alternate_phone: '',
-        notes: '',
-        type: 'person',
-        reported_date: 1478469976421
-      },
-      name: 'Number three district',
-      external_id: '',
-      type: 'district_hospital'
-    }
   }
 ];
 
 const savedUuids = [];
+const saveDocs = function(done, documents) {
+  browser.ignoreSynchronization = true;
+  protractor.promise
+    .all(documents.map(utils.saveDoc))
+    .then(results => {
+      results.forEach(result => {
+        savedUuids.push(result.id);
+      });
+    })
+    .then(() => utils.getDoc(userSettingsDocId))
+    .then((user) => {
+      user.contact_id = contactId;
+      return utils.saveDoc(user);
+    })
+    .then(done)
+    .catch(err => {
+      console.error('Error saving docs', err);
+      done(err);
+    });
+};
+
+
+
+const selectRadioButton = (value) => {
+  element(by.css(`[value=${value}]`)).click();
+};
 
 module.exports = {
   configureForm: done => {
-    browser.ignoreSynchronization = true;
-    protractor.promise
-      .all(docs.map(utils.saveDoc))
-      .then(results => {
-        results.forEach(result => {
-          savedUuids.push(result.id);
-        });
-      })
-      .then(() => utils.getDoc(userSettingsDocId))
-      .then((user) => {
-        user.contact_id = contactId;
-        return utils.saveDoc(user);
-      })
-      .then(done)
-      .catch(err => {
-        console.error('Error saving docs', err);
-        done(err);
-      });
+    saveDocs(done, docs);
+  },
+
+  saveDocs: (done, documents) => {
+    saveDocs(done, documents);
```

Comment:
Seems weird that you would need to redefine this here, but if you really have to then this could likely be rewritten:
```
saveDocs: saveDocs,
```